assembl/static2/js/app/components/common/tree/tree.jsx

```diff
@@ -53,6 +50,16 @@ class Tree extends React.Component<Props> {
     }
   }
 
+  componentDidUpdate() {
+    this.triggerScrollToRow(this.props.initialRowIndex);
```

Comment:
I would use `prevProps` (first argument) and compare to `this.props` to check if `initialRowIndex` changed before to scroll

Reply:
good point